### ISSUE TITLE
Release available?

### ISSUE THREAD

**Issue by Moult** [tercoide/GauchoCAD#42]
Is there somewhere where I can download a release for any platform? Or installation / compilation instructions? (I'm on Linux personally)

I saw this software from the libredwg mailing list, and would love to help improve libredwg support.

**Comment by tercoide** [comment]
Hello Dion, at this point there is no releases. But you can clone the 
code from git and run it from Gambas3 if you want. We are in early alpha 
stage at this point and using the libredwg from the command line.

We are developing in Linux too. The steps to use it are be:

1-Get Gambas3 from

http://gambas.sourceforge.net/en/main.html

2-Clone the code from github

3-Open the project in Gambas3 and Run it.

We are happy to welcome you to the team.

Please contact me or Tincho for further help.

Martin

**Comment by belmotek** [comment]
Hi Dion,
Thanks you for your interest in the project, I hope you join the team.
I took a look at your code on github and I see that you are familiar with standard IFC, good point, I also saw that you use Gentoo linux, right?
I tell you some considerations to keep in mind depending on what you want to do with or in GauchoCAD.
If you want to work in:
1) The bound between LibreDWG and Gambas3. Here you need the Gambas3 source code, which can be the development version or the stable version (3.15.2). In this case you don't need the GauchoCAD source code.

2) Development of GauchoCAD. Here you need the GauchoCAD source code, Martin C. takes care of graphic where with the Gambas3 OpenGL component. then I take care of the GUI and the reading of DWG files that at the moment I use SHELL with a program called **dwgread** and we use the JSON format. so far it's read-only.

3) Use GauchoCAD: We have not yet created any installation package because it is an alpha aversion, so there are two options either you run it from Gambas3 IDE or you create a package for your distribution and install it on your system ( I don't recommend it yet).
Yo need here to install the IDE. I don't know if Gambas3 IDE exists in the Gentoo repositories but in that case you can install the IDE from sources.
http://gambas.sourceforge.net/en/main.html#

Note: To install LibreDWG you have to download the source code, compile it and install it. In the web page they explain how to do it but I leave you the script that I use in case it is useful to you.
https://gambas-es.org/showthread.php?tid=151&highlight=libredwg
Regards.

**Comment by Ebm2003NYC** [comment]
Hi,

I really want to try your software, how I can on Windows 10?!

Thank you,

**Comment by Moult** [comment]
Hi @belmotek ! Unfortunately I don't have the time to dedicate to another free software project, but I'm always on the look-out for free software that can realistically challenge the proprietary tools like AutoCAD in the architecture, engineering, and construction industry. When I have some spare time, I will take your advice and try to run GauchoCAD from the IDE.

**Comment by belmotek** [comment]
Yes, don't worry, the time one has for these things is limited, but thanks for your interest.
Tell me, what project are you involved in?
Regards.

**Comment by belmotek** [comment]
> Hi,
> I really want to try your software, how I can on Windows 10?!
Hi, Ebm2003NYC, the software run on linux, so you need to install it in a virtual machine.
Regards.

**Comment by Moult** [comment]
@belmotek I am involved in IfcOpenShell, the BlenderBIM Add-on, Ladybug-Blender, and OSArch:

 - https://github.com/IfcOpenShell/IfcOpenShell
 - https://ifcopenshell.org
 - https://blenderbim.org
 - https://github.com/ladybug-tools/ladybug-blender
 - https://osarch.org/

**Comment by belmotek** [comment]
@Moult That seems enough. :-)
